src/elements/content-sidebar/activity-feed/activity-feed/AnnotationActivityLinkProvider.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import AnnotationActivityLink from '../annotations/AnnotationActivityLink';
+
+import annotationsMessages from '../annotations/messages';
+
+import type { Annotation } from '../../../../common/types/feed';
+
+type AnnotationActivityLinkProviderProps = {
+    isCurrentVersion: boolean,
+    item: Annotation,
+    onSelect: (annotation: Annotation) => void,
+};
+
+const AnnotationActivityLinkProvider = ({ isCurrentVersion, item, onSelect }: AnnotationActivityLinkProviderProps) => {
+    const { file_version, id, target } = item;
+
+    const isFileVersionUnavailable = file_version === null;
```

Comment:
How about if file_version is undefined or 0? Should we just use a negation instead?

Reply:
Here (and below) I just copied the logic that was already being used for creating AnnotationActivityLink. That said, it could be worth reviewing. I'll take a look at this :)

Reply:
It's because the type for `file_version` on the `Annotation` type is `file_version: BoxItemVersionMini | null`.

Reply:
(It can't be undefined or 0.)